frontend/pages/journey/[id].vue

```diff
@@ -0,0 +1,441 @@
+<script setup lang="ts">
+import {
+  compareAsc,
+  differenceInDays,
+  format,
+  intervalToDuration,
+} from "date-fns";
+import QRCode from "qrcode";
+import JSConfetti from "js-confetti";
+const route = useRoute();
+const journeyId = route.params.id;
+const qrcode = ref("");
+const duringJourney = ref(false);
+const journeyEnded = ref(false);
+const day = ref(0);
+const tensDays = ref(0);
+const hundredsDays = ref(0);
+const jsConfetti = new JSConfetti();
+
+definePageMeta({
+  middleware: ["sanctum:auth"],
+});
+interface Journey {
+  name: string;
+  invite: string;
+  destination: string;
+  from: string;
+  to: string;
+}
+
+const journeyData = ref({} as Journey);
+
+const client = useSanctumClient();
+await client(`/api/journey/${journeyId}`, {
```

Comment:
what happens if journey can't be loaded?
-> now an empty page

add error handly, e.g. nuxt error i think